.github/contributing.md

```diff
@@ -163,3 +164,32 @@ each package to suit the package's requirements.
 We use GitHub actions to manage our CI pipeline.
 
 The workflows can be found in `.github/workflows`
+
+## Reporting Issues
+
+If you encounter any issues or have a feature request, please [create a new issue](https://github.com/interledger/rafiki/issues/new/choose) and provide the following details:
+
+- A clear and descriptive title.
+- A detailed description of the issue, including steps to reproduce if applicable.
+- Information about your environment (e.g., operating system, browser, version).
+- Any relevant screenshots or error messages.
+
+## Submitting Pull Requests
+
+1. [Fork](https://github.com/interledger/rafiki) the repository.
+2. Create a new branch from `main`.
+3. Make your changes and commit them.
+4. Create a pull request (PR) to `main`.
+5. Ensure your PR includes a clear title and description.
```

Comment:
we need to add a line here that talks about our PR title requirements for conventional commits: https://www.conventionalcommits.org/en/v1.0.0/

Reply:
Added.